fluentd-modsecurity/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM alpine/git:v2.24.1 as builder
+
+WORKDIR /
+RUN git clone https://github.com/bitsofinfo/fluentd-modsecurity.git
+
+FROM fluent/fluentd:v1.9.3-debian-1.0
+
+# Use root account to use apt
+USER root
```

Comment:
Is `fluent/fluentd:v1.9.3-debian-1.0` using another user ? If yes, can you document it ?

Reply:
Fait